src/main/resources/pdfconfig/letterhead.html

```diff
@@ -0,0 +1,5 @@
+<div th:fragment="letterhead">
+    <h2> Welcome to [[${data.getBusinessname()}]]! We are participating in [[${data.getTown()}]], [[${data.getState()}]]'s
```

Comment:
how are we gonna be getting the town or the state if we do not store them and no api command accepts them?

Reply:
we are storing a full address, but more importantly we aren't parsing it. not everywhere in the world has states anyway. i assume this is just temporary copy until product figures out exactly what to put here.